plugins/Toolbox/src/Toolbox.py

```diff
@@ -540,9 +557,7 @@ def getTotalNumberOfMaterialPackagesByAuthor(self, author_id: str) -> int:
 
     @pyqtSlot(str, result = bool)
     def isEnabled(self, package_id: str) -> bool:
-        if package_id in self._plugin_registry.getActivePlugins():
-            return True
-        return False
+        return package_id in self._plugin_registry.getActivePlugins()
```

Comment:
This is just refactor, doesn't relate to the ticket.

Reply:
It's still a valid change. No idea why it was like this in the first place.